dwbb/src/reference/DW01_add.scala

```diff
@@ -3,7 +3,7 @@
 package oscc.dwbb.reference
 
 import chisel3._
-import oscc.dwbb.interface._
+import org.chipsalliance.dwbb.interface.DW01_add
 
 class DW01_add(parameter: DW01_add.Parameter) extends ReferenceModule(new DW01_add.Interface(parameter), parameter) {
```

Comment:
I'm considering rename `ReferenceModule` to DUT... It can be done in the future PRs.

Reply:
That's what I thought, the actual "reference modules" here are Verilog models.